devel.yaml

```diff
@@ -51,6 +53,9 @@ services:
         extends:
             file: common.yaml
             service: smtpfake
+        ports:
+            - "127.0.0.1:1025:1025"
```

Comment:
@Yajo think you don't need this. only http interface to 8025 is needed. 1025 port is internal.

Reply:
Yeah, in my mind it seemed useful, so we'd be able to connect i.e. Outlook (which you know how badly renders emails) to the local port and see the email, but actually it's just an SMTP port (not IMAP/POP3) so it's useless as you say.